python/tank/platform/engine.py

```diff
@@ -1742,6 +1761,59 @@ def _define_qt_base(self):
 
         return base
 
+    def __define_qt5(self):
+        """
+        This will be called at initialization to discover every PySide 2 modules, if available.
```

Comment:
oops just saw it in the code comments below - nice! Wonder if a brief single line in the docstring could be useful too?